src/main/java/com/college/erpmgmt/entities/Student.java

```diff
@@ -1,19 +1,17 @@
 package com.college.erpmgmt.entities;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+
+import javax.persistence.*;

```

Comment:
don't import all until it is required or the import is more than 30. this has been automatically done by intellij, but you can change the behaviour on when to use star.
do the below setting on intellij

![image](https://user-images.githubusercontent.com/54737713/162149034-5047a784-e860-4b13-ba06-0b5a6d04f13a.png)